.travis.yml

```diff
@@ -24,9 +24,8 @@ cache:
 
 script:
   # need to override as the default is to test
-  - sbt 'set credentials += Credentials("whitesource", "whitesourcesoftware.com", "", System.getenv("WHITESOURCE_KEY"))'
   - sbt -J-XX:ReservedCodeCacheSize=256m +update
-  - if [[ "$TRAVIS_BRANCH" == "master" ]]; then sbt whitesourceCheckPolicies; fi
+  - if [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_EVENT_TYPE" == "push" ]]; then sbt 'set credentials += Credentials("whitesource", "whitesourcesoftware.com", "", System.getenv("WHITESOURCE_KEY"))' whitesourceCheckPolicies whitesourceUpdate; else sbt 'set credentials += Credentials("whitesource", "whitesourcesoftware.com", "", System.getenv("WHITESOURCE_KEY"))' whitesourceCheckPolicies; fi 
```

Comment:
I guess I'm missing something, can we go over the process at some point? Only on push to master sounds a bit weird - does merging to master in github count then? If yes, then I get it - so we only update with things that we actually merged, and not random things that people PR right? Or did I misinterpret this?